.github/workflow/linters.yml

```diff
@@ -0,0 +1,100 @@
+Skip to content
+Search or jump to…
+Pull requests
+Issues
+Marketplace
+Explore
+ 
+@davmba 
+microverseinc
+/
+linters-config
+Public
+Code
+Issues
+21
+Pull requests
+7
+Actions
+Projects
+Wiki
+Security
+Insights
+linters-config/html-css/.github/workflows/linters.yml
+@sinansevgi
+sinansevgi Update webhint to version 7
+Latest commit 39a68f5 7 days ago
+ History
+ 7 contributors
+@SpaYco@MauricioRobayo@sinansevgi@Meltrust@MacCrazyman@nidalaa@bolah2009
+56 lines (53 sloc)  1.91 KB
```

Comment:
- [ ] You should not modify the linter config file, kindly restore the linters.yml to the original state so the linters are working correctly